app/src/main/res/values-land/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="add">ADD</string>
+    <string name="name">name</string>
+    <string name="parameter_without">parameter (without %)</string>
+</resources>
```

Comment:
Why a "different" value for landscape mode? If you don't specify it again, the default value will be used anyway.